pykechain/models/property_reference.py

```diff
@@ -331,3 +331,99 @@ def upload(self, data: Any) -> None:
             filepath=data,
         )
         self.value = self.value + [stored_file] if self.value else [stored_file]
+
+
+class SignatureProperty(_ReferenceProperty):
```

Comment:
Why doesn't this inherit the `StoredFilesReferencesProperty`? I think that should be the start, most functions are already developed there, so this is just code duplication

Reply:
I started with `StoredFIlesReferencesProperty` and found out that functions needed to be redeveloped as it handles lists of Stored files. So I took what what in this Storedfiles thing and replicated this on top of the `_ReferenceProperty` base class.

However due to your comment I will revisit this once more.